about.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ABOUT | Flavors of Peru 2022</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="about-body">
+    <div id="navbar-menu" class="backdrop display-none">
+        
+        
+        <ul class="menu-items">
+
+            <li class="exit-list"><a  id="exit-link" class="enu-list-option" href="#">X</a>  </li>
+            <li class="menu-list-option"><a href="index.html" class="menu-option">Home</a></li> 
+            <li class="menu-list-option"><a href="about.html" class="menu-option">About</a></li> 
+            <li class="menu-list-option"><a href="index.html#program" class="menu-option">Program</a></li> 
+            <li class="menu-list-option"><a href="index.html#speakers" class="menu-option">Speakers</a></li> 
+            <li class="menu-list-option"><a href="index.html#submit-button" class="menu-option">Join</a></li> 
+        </ul>
+
+    </div>
+    <header id="header" class="about-header header">
+           
+        <div id="navbar-container">
+
+            <nav id="desktop-navbar">
```

Comment:
- [ ] When I open the navbar hamburger button and then click X the hamburger button dissapears. So cannot open it again without refreshing the page. Kindly fix this
![Screenshot from 2021-10-14 22-40-48](https://user-images.githubusercontent.com/62705947/137392881-d181ede7-7bff-4103-8f0b-e0f80ec3109a.png)

Reply:
:heavy_check_mark: Well resolved